content/blog/2022/12/kubewarden-2022-wrapped.md

```diff
@@ -0,0 +1,119 @@
+
+By being part of CNCF, Kubewarden also got access to another CNCF project,
+[CLOMonitor](https://clomonitor.io/projects/cncf/kubewarden).
+Thanks to CLOMonitor, Kubewarden's users have a clear picture of the project health
+and can assess its development and security practices.
+
+## Pod Security Policy replacement
+
+During the month of August, Kubernetes 1.25 release finally removed
+[Pod Security Policies](https://kubernetes.io/docs/concepts/security/pod-security-policy/).
+
+This event was planned and advertised for a long time. Due to this,
+the Kubewarden developers were able to provide
+[tooling, policies, and documentation](https://docs.kubewarden.io/tasksDir/psp-migration)
+to smoothen the migration process from Kubernetes Pod Security Policies to
+Kubewarden policies.
+
+## Secure supply chain
+
+The topic of secure supply chain started to emerge near the end of 2021, but it
+was during this year that many projects and organizations tackled it.
+
+The [sigstore](https://sigstore.dev) project proved to be one of the pillars of
+these initiatives.
+Kubewarden invested early in the Sigstore project, with some of our maintainers being
+active also within the Sigstore community.
+
+During the last year Kubewarden embraced Sigstore to
+[securely distribute its own assets](/blog/2022/04/securing-kubewarden-policies/),
+like container images, policies, helm charts and cli utilities.
+Moreover, we exposed Sigstore capabilities to our policy authors and provided
+a [ready to use policy](https://artifacthub.io/packages/kubewarden/verify-image-signatures/verify-image-signatures)
+that can be used to vet the container images used by Kubernetes workloads.
+
+Sigstore provides different ways to sign artifacts: with
+traditional keys,
+[keyless mode](/blog/2022/05/verifying-image-signatures/),
+[keyless mode from a GitHub Action workflow](/blog/2022/07/verify-signatures-with-gha-and-prefix/) and
+[self-signed certificates](/blog/2022/12/sigstore-certificate-verification/)
+stored on a physical token or provided by a KMS.
+During the last year we kept extending Kubewarden to ensure all these scenarios would be
+available also to our users.
+
+## Other notable features
+
+A stream of improvements have been made to the Kubewarden project during the last
+year.
+
+The Kubernetes integration was further expanded by introducing the new
+[`AdmissionPolicy`](/blog/2022/03/admission-policy/) Custom Resource. This allows
+regular Kubernetes users to define policies that are scoped to a specific
+Namespace.
+
+Next, we introduced a new operating mode for all our policies: the
+["monitor"](/blog/2022/05/monitor-mode/) mode. This allows Kubernetes administrators
+to better understand how a policy would impact their existing clusters once being
+enforced.
+
+More recently, the Kubewarden developers simplified the process of deploying
+Kubewarden inside of [air gapped](/blog/2022/11/airgap/) environments.
```

Comment:
Do we want to provide a line or two on how we did this? It seems a bit abrupt.
```suggestion
Kubewarden inside of [air gapped](/blog/2022/11/airgap/) environments.
```

Reply:
I fear the article is already pretty long. The plan is to have the reader take a look at the linked blog post to learn more about that.

If you are against that, I can add a few lines like: 

```md
More recently, the Kubewarden developers simplified the process of deploying
Kubewarden inside of [air gapped](/blog/2022/11/airgap/) environments. This
has been achieved by providing small helpers, writing documentation and, most
important of all, dedicated QA efforts.
```

Reply:
That one line is enough IMO. The last line ended very abruptly which is why it felt like there needed to be something.

Reply:
Added